src/test/scala/heuristics/ClassificationTest.scala

```diff
@@ -0,0 +1,60 @@
+package heuristics
+
+import extractor.SemanticCtx
+import framework.SemanticdbTest
+import org.langmeta.semanticdb.Synthetic
+
+class ClassificationTest extends SemanticdbTest {
+
+
+  case class Wrappers(extensions: Seq[Synthetic], conversions: Seq[Synthetic])
+  /**
+    * A function that tries to classify the usage of a wrapper function, in either
+    * a class extension, an implicit conversion, or other.
+    * @param wrapperFunctions
+    * @return
+    */
+  def classifyUsages(ctx: SemanticCtx, wrapperFunctions: Seq[Synthetic]): Wrappers = {
```

Comment:
Not sure I understand this? Why is this in a test?

Reply:
Those (everything in `heuristics`) are exploratory tests, not really testing any functionality in the extractor, but exploring and demonstrating different classification techniques we can reimplement later. They are in the tests, because they provide some way to test our classification logic in executable code.

That said, if you are confused it means that the comment I wrote wasn't clear enough, I'll re-document it.